### ISSUE TITLE
Remove signature for new intent-types (new-version/cloned)

### ISSUE THREAD

**Issue by wisotzky** [nokia/vscode-intent-manager#23]

**Comment by wisotzky** [comment]
We've unblocked the modification of tags in the `meta-info.json` if the intent-type is signed.

Will need to revisit NSP product evolution 24.8 and beyond, when we enforce write protection at engine-level.
The general desire would be the following:
- On systems classified as "production", the only way to install/update intent-types is via CAM.
- On systems classified as "production", only signed intent-types are allowed.
- If customer has not licensed "programmable automation", installation of self-signed intent-type artifacts is blocked
- Modification of signed intent-types is blocked. CAM must be used to update those.

In conclusion, editing intent-types is bound to dev/lab/test environments.
